javascript-typing-tutor/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Javascript Typing Tutor</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Arvo:wght@700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="row">
+      <h1 class="h1-box column-full">
+        <span class="span-neutral ">g</span>
+        <span>r</span>
+        <span>u</span>
+        <span>m</span>
+        <span>p</span>
+        <span>y</span>
+        <span class="space">&nbsp</span>
```

Comment:
You should be able to do this with just normal spaces, no non-breaking spaces should be necessary.